examples/p4est_3d_dgsem/elixir_advection_basic.jl

```diff
@@ -1,21 +1,23 @@
+# The same setup as tree_3d_dgsem/elixir_advection_basic.jl
+# to verify the P4estMesh implementation against TreeMesh
 
 using OrdinaryDiffEq
 using Trixi
 
 ###############################################################################
 # semidiscretization of the linear advection equation
 
-advection_velocity = (1.0, 1.0, 1.0)
+advection_velocity = (0.2, -0.7, 0.5)
 equations = LinearScalarAdvectionEquation3D(advection_velocity)
 
 # Create DG solver with polynomial degree = 3 and (local) Lax-Friedrichs/Rusanov flux as surface flux
 solver = DGSEM(polydeg=3, surface_flux=flux_lax_friedrichs)
 
-coordinates_min = (-1.5, -0.9, 0.0) # minimum coordinates (min(x), min(y), min(z))
-coordinates_max = ( 0.5,  1.1, 4.0) # maximum coordinates (max(x), max(y), max(z))
+coordinates_min = (-1.0, -1.0, -1.0) # minimum coordinates (min(x), min(y), min(z))
+coordinates_max = ( 1.0,  1.0,  1.0) # maximum coordinates (max(x), max(y), max(z))
 
-# Create P4estMesh with 8 x 10 x 16 elements (note `refinement_level=1`)
-trees_per_dimension = (4, 5, 8)
+# Create P4estMesh with 8 x 8 x 8 elements (note `refinement_level=1`)
+trees_per_dimension = (4, 4, 4)
```

Comment:
Are there still setups that do not use the same number of trees in all dimensions?

Reply:
No

Reply:
OK. I think the next time such a test is added that does not exactly reproduce another setup, we should make sure that we use not such a trivial example. Same number in all directions *plus* power of two is one of those special configurations that annoying bugs can hide behind...